sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit git-r3 readme.gentoo-r1
+
+DESCRIPTION="Raspberry PI boot loader and firmware"
+HOMEPAGE="https://github.com/raspberrypi/firmware"
+SRC_URI=""
+
+LICENSE="GPL-2 raspberrypi-videocore-bin"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+DEPEND=""
```

Comment:
Please remove `DEPEND` and `RDEPEND`. They are not used anyway.

Reply:
Done.